src/UseCases/AddComment/AddCommentUseCase.php

```diff
@@ -46,6 +51,10 @@ public function addComment( AddCommentRequest $addCommentRequest ): AddCommentRe
 
 		$donation->addComment( $this->newCommentFromRequest( $addCommentRequest ) );
 
+		if ( !$this->commentTextPassesValidation( $addCommentRequest->getCommentText() ) ) {
```

Comment:
As far as I understood the code, the DNS entries are not checked. It's checking URLs, but not doing the DNS check.